service/gcs/bridge/bridge.go

```diff
@@ -90,43 +90,43 @@ func (b *bridge) loop() error {
 		var response interface{}
 		switch header.Type {
 		case prot.ComputeSystemCreateV1:
-			utils.LogMsg("received from HCS: ComputeSystemCreateV1")
+			logrus.Print("received from HCS: ComputeSystemCreateV1")
```

Comment:
I'm more used to seeing logrus.Info than logrus.Print (which I think are equivalent but I had to look it up).